### PR TITLE
Seed AlphaZero Threads

### LINKED ISSUE #401: Alpha Zero plays same games on each thread.
Each thread is run with the same seed which means that all the games are the same.
This is then the same as running it in single threaded, but it could also bias the network.

### PR DESCRIPTION
**Pull request by goshawk22**
Fixes #401 where each thread will play the same games.

### CODE REVIEW

File: benchmark/torch/AlphaZero/Coach.py [hunk 1]
```diff
@@ -54,9 +54,9 @@ def __init__(self, game, args):
 
         self.test_dataset = get_test_dataset()
 
-    def _run_remote_tasks(self, signal_queue):
+    def _run_remote_tasks(self, signal_queue, i):
```
Comment (TomorrowIsAnOtherDay) [inline benchmark/torch/AlphaZero/Coach.py]: please rename `i` to `seed`.

File: benchmark/torch/AlphaZero/actor.py [hunk 2]
```diff
@@ -23,7 +23,8 @@
 
 @parl.remote_class
 class Actor(object):
-    def __init__(self, game, args):
+    def __init__(self, game, args, i):
```
Comment (TomorrowIsAnOtherDay) [inline benchmark/torch/AlphaZero/actor.py]: Please rename `i` to `seed`.

**Comment by goshawk22** [comment]
Updated PR in line with comments.

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
LGTM.

**Pull request merged**